### PR TITLE
Code refactor and Rabbitmq

### PR DESCRIPTION
**Pull request by AhmedSoliman7**

### CODE REVIEW

File: src/main/java/com/linkedin/replica/recommender/messaging/ClientMessagesReceiver.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.concurrent.TimeoutException;
+
+public class ClientMessagesReceiver {
+    private Configuration configuration = Configuration.getInstance();
+    private RecommendationService notificationService = new RecommendationService();
+    private final String QUEUE_NAME = configuration.getAppConfig("rabbitmq.queue.client");
+    private final String RABBIT_MQ_IP = configuration.getAppConfig("rabbitmq.ip");
```
Comment (HZahran) [inline src/main/java/com/linkedin/replica/recommender/messaging/ClientMessagesReceiver.java]: no rabbitmq.queue.client or rabbitmq.ip in app.config file !

File: src/main/java/com/linkedin/replica/recommender/messaging/ClientMessagesReceiver.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import com.rabbitmq.client.*;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.concurrent.TimeoutException;
+
+public class ClientMessagesReceiver {
+    private Configuration configuration = Configuration.getInstance();
+    private RecommendationService notificationService = new RecommendationService();
```
Comment (HZahran) [inline src/main/java/com/linkedin/replica/recommender/messaging/ClientMessagesReceiver.java]: matsameeha recommendationService :"D

Comment (omarradwan) [reply src/main/java/com/linkedin/replica/recommender/messaging/ClientMessagesReceiver.java]: Hahahahhahaah

**Comment by HZahran** [comment]
Reviewed

**Comment by HZahran** [comment]
3ayzeen te3melo 7aga tany walla 5alas ?

**Comment by omarradwan** [comment]
I guess 5alas

**Comment by HZahran** [comment]
tah tamam (Y)

**Pull request merged**